iceoryx_posh/include/iceoryx_posh/internal/roudi/service_registry.hpp

```diff
@@ -31,6 +31,7 @@ namespace iox
 {
 namespace roudi
 {
+static capro::ServiceDescription serviceRegistryService{"ServiceRegistry", "RouDi_ID", "ServiceRegistry"};
```

Comment:
Please never use static globals otherwise we run again into the C++ static initialization order fiasco. Please use the `iceoryx_posh_types.hpp` and store there 3 constexpr `capro::IdStrings` with the service name.